### PR TITLE
Added editUser function and added paidMember field

### PR DESCRIPTION
**Pull request by jeffjramos21**
Added editUser to services, controller, and index. Added paidMember field to User model.

### CODE REVIEW

File: functions/src/Services/User.ts [hunk 1]
```diff
@@ -1,10 +1,39 @@
...
+/**
+ * Handles retrieving the user collection with the converter.
+ */
+function getUserCollection() {
+	return firestore().collection('users').withConverter(userConverter);
+}
 
 export const createUser = async (user: User) => {
-	const userCollection = db.collection('users').withConverter(userConverter);
+	const userCollection = getUserCollection().withConverter(userConverter);
```
Comment (BautistaBertolami) [inline functions/src/Services/User.ts]: you don't need to use withConverter() here since getUserCollection() already calls it.

**Review by BautistaBertolami — CHANGES_REQUESTED** [review]

File: functions/src/Services/User.ts (hunk 1 above)
Comment (jeffjramos21) [reply functions/src/Services/User.ts]: fixed

File: functions/src/Models/User.ts [hunk 2]
```diff
@@ -6,6 +6,7 @@ import firebase from 'firebase-admin';
  */
 export class User {
 	voted: boolean = false;
+
```
Comment (idelmr) [inline functions/src/Models/User.ts]: Remove empty line

Comment (jeffjramos21) [reply functions/src/Models/User.ts]: done

**Review by BautistaBertolami — APPROVED** [review]

File: functions/src/Services/User.ts [hunk 3]
```diff
@@ -1,10 +1,39 @@
 import { User, userConverter } from '../Models/User';
-import { db } from '../index';
+import { firestore } from 'firebase-admin';
```
Comment (smontana30) [inline functions/src/Services/User.ts:2]: why are you import firestore db is firestore. you are basic doing more work also db is the firestore we initialized with the app. So i think you will need to change it back.

File: functions/src/Services/User.ts [hunk 4]
```diff
@@ -1,10 +1,39 @@
...
+
+/**
+ * Handles retrieving the user collection with the converter.
+ */
+function getUserCollection() {
+	return firestore().collection('users').withConverter(userConverter);
```
Comment (smontana30) [inline functions/src/Services/User.ts:8]: u can make this db().collection('users').withCon

File: functions/src/Services/User.ts (hunk 1 above)
Comment (smontana30) [inline functions/src/Services/User.ts]: why are you call withConverter() twice you already call it in the getUserCollection() method

File: functions/src/Services/User.ts [hunk 5]
```diff
@@ -1,10 +1,39 @@
...
+		applied: user.applied,
+		userCommittees: user.userCommittees,
+		color: user.color,
+		flag: user.flag,
+		picture: user.picture,
+		email: user.email,
+		major: user.major,
+		country: user.country,
+		gender: user.gender,
+		birthday: user.birthday })
```
Comment (smontana30) [inline functions/src/Services/User.ts]: put the "})" on a new line this look ugly

**Review by smontana30 — APPROVED** [review]

**Pull request merged**